### ISSUE TITLE
Introducing language specific isFrontPage

### ISSUE THREAD

**Issue by henrikwirth** [valu-digital/wp-graphql-polylang#5]
My suggestion is to introduce some identifier for the frontpage of each language.

This would help to identify the Homepage of each language on the StaticSiteGeneration-Side, so one can adjust the paths to be `/`, `/en/`, `/de/ without the title like `/en/home`.

**Example Query could be:**

```
query MyQuery {
  pages(where: {parent: null}, first: 100) {
    nodes {
      title
      uri
      language {
        isFrontPage
      }
    }
  }
}
```

@epeli suggested to do something like this in the core wp-graphql:

```
return apply_filters( 'graphql_post_is_front_page', $post->ID === get_option('page_on_front'), $post);
```

And wp-graphql-polylang could extend that by returning true for all translated front pages too.

**Comment by MKlblangenois** [comment]
@esamattis any idea about this issue ?

**Comment by henrikwirth** [comment]
I think this is part of the WPGraphQL core now actually.

**Comment by adriandmitroca** [comment]
I don't agree, front page for certain languages are handled on top of Polylang plugin, which is not directly core of WordPress itself.

**Comment by esamattis** [comment]
Implemented on top of the core `isFrontPage` in https://github.com/valu-digital/wp-graphql-polylang/pull/48 in released with v0.6.0
